tests/Makefile

```diff
@@ -21,6 +21,7 @@ TESTS := \
 	file_create \
 	file_delete \
 	file_rename \
+	file_permission \
```

Comment:
If we are listing the tests in alphabetical order we should add the `file_permission` test *before* the `file_rename` test.